src/SchemaRegistry.ts

```diff
@@ -33,7 +34,7 @@ interface RegisteredSchema {
 interface Opts {
   compatibility?: COMPATIBILITY
   separator?: string
-  subject?: string
+  subject: string
```

Comment:
is this change necessary? wouldn't it be backwards incompatible for TS users who use Avro without specifying a subject?